util/perl/TLSProxy/Proxy.pm

```diff
@@ -294,6 +294,7 @@ sub clientstart
     #Wait for either the server socket or the client socket to become readable
     my @ready;
     my $ctr = 0;
+    $SIG{PIPE} = "IGNORE";
```

Comment:
Make that `local $SIG{PIPE} = "IGNORE"`. If a caller has a different idea what it should be, you don't want to disturb that.

Reply:
ok.